modules/free/arrow-free-data/src/main/kotlin/arrow/free/StackSafeMonadContinuation.kt

```diff
@@ -49,18 +49,6 @@ open class StackSafeMonadContinuation<F, A>(M: Monad<F>, override val context: C
   }
 }
 
-@Deprecated(
```

Comment:
This function doesn't exist in 0.9.0 stable, do we agree on just deleting it. It originated in "Next".

Reply:
I'm surprise it didn't exist. Maybe as a member function?

Reply:
Maybe.., but deleting it is fine right?

Reply:
I think we agreed in deleting this as part of the binding rework and decide to remove it because it was hard to refactor. We are gonna support this natively with plugins for all comprehensions that are flagged as stack safe without the Free runtime using suspension and a small loop.